source/_posts/2019-01-14-Google-Home.md

```diff
@@ -0,0 +1,56 @@
+---
+title: A Guide to Google Home Integrations
+date: 2019-01-14 10:00:00
+tags: server,google,home,ai,api,cloud,serverless
+authorIds: 
+- brto
+categories:
+- Vapor
+---
+
+A while back I was tasked with integrating DMI’s weather service into Google Home. The goal of the service was to  enable users to ask:  “Hey DMI, what is the weather.” As well as: “Hey DMI, should I bike to work today?” Given my only experience with Google Home was through its advertisements, I went in with the expectations that integration would be quick and painless—just configure a few interests, link it with DMI’s API and start checking the weather. Surprisingly, this was far from actuality. Configuration of the app, dialogues and Cloud Functions is split across 3 different services. The initial clunkiness aside, the services seemed to work well and the tooling was relatively stable. Training intents in DialogFlow was a seamless and enjoyable experience, it just worked. Using the website’s inline code editor, I was able to develop and deploy the fulfillment to Google Cloud Functions.
```

Comment:
Do people outside Denmark know what DMI is? Maybe just a link and a short mention of what it is the first time you mention it